src/components/annotations/AnnotationsList.vue

```diff
@@ -12,11 +12,21 @@
       ]"
       @click="isText(annotation) ? ()=>{} : toggle(annotation)"
     >
-      <div class="t-flex t-items-center t-space-x-2">
-        <AnnotationIcon v-if="!isText(annotation)" :name="getIconName(annotation.body['x-content-type'])" />
+      
+        <div v-if="!isVariant(annotation)" class="t-flex t-items-center t-space-x-2"> 
```

Comment:
Please create a new component `AnnotationVariantItem` and move all the logic there.